CONVENTIONS.md

```diff
@@ -122,6 +122,18 @@ Write quick descriptions for parameter javadocs as well as `@return` tags, with
 
 Use `@link`, `@linkplain` and `@see` tags to refer to other parts of the code.
 
+### Game Content Capitalization
```

Comment:
```suggestion
### Game content capitalization
```

Reply:
Fixed in d68ecf1db82b231fc59473fc8625d9ad03b811ba